celery/backends/asynchronous.py

```diff
@@ -200,6 +200,7 @@ def _wait_for_pending(self, result,
         return self.result_consumer._wait_for_pending(
             result, timeout=timeout,
             on_interval=on_interval, on_message=on_message,
+            **kwargs,
```

Comment:
Do we have test to cover the kwargs param?

Reply:
I can't seem to find any existing tests for `kwargs` for `AsyncBackendMixin._wait_for_pending`, but `BaseResultConsumer._wait_for_pending` supports `kwargs` (which also doesn't seem to have tests)